### ISSUE TITLE
“Armor Making II” tech should depend on “Rubber 1” and “Leaded Brass”

### ISSUE THREAD

**Issue by VoidPhantom** [Factorio-Xander-Mod/Xander-Mod#33]
Because heavy armor requires brass plates and vulcanized rubber to craft, this should be reflected in the tech tree. Also, if this is done, then “Armor Making III” ’s explicit dependency on “Rubber 1” should be removed.

**Comment by purdueme91** [comment]
I agree with this one.  You unlock armor II very early then are unlocking tech after tech to get to brass.
